### PR TITLE
[release-2.2] Organization rename to `stolostron`

### PR DESCRIPTION
**Pull request by dhaiducek**
- https://github.com/stolostron/backlog/issues/18776

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -22,3 +22,5 @@ replace (
 	howett.net/plist => github.com/DHowett/go-plist v0.0.0-20181124034731-591f970eefbb
 	k8s.io/client-go => k8s.io/client-go v0.17.4 // Required by prometheus-operator
 )
+
+replace github.com/open-cluster-management/api => open-cluster-management.io/api v0.0.0-20200610161514-939cead3902c
```
Comment (ycao56) [inline go.mod:26]: Merge this replace with above. Build failed. You may need to replace the client-go version to `v0.18.3`

File: .travis.yml [hunk 2]
```diff
@@ -53,10 +53,10 @@ jobs:
           make component/push
           if [ "$IMAGE_SCAN" != "false" ]; then make security/scans; fi;
         - if [ "$TRAVIS_PULL_REQUEST" == "false" -a "$TRAVIS_BRANCH" == "master" ]; then
```
Comment (ycao56) [inline .travis.yml:55]: "$TRAVIS_BRANCH" == "release-2.2"

File: go.mod (hunk 1 above)
Comment (ycao56) [reply go.mod:26]: That was the API change between 0.17 and 0.18

**Comment by ycao56** [comment]
@dhaiducek can you check why build failed?

**Comment by dhaiducek** [comment]
Yeah, it's because we've never pushed a `latest-2.2` tag to `stolostron` in Quay. I'm setting the tag for now to this PR and then I'll follow it up with a PR to change it to `latest-2.2` (assuming it works).

**Comment by dhaiducek** [comment]
This PR is ready for review.

**Review by ycao56 — APPROVED** [review]

**Pull request merged**